src/renderer/components/App.tsx

```diff
@@ -64,6 +66,19 @@ export default class App extends React.Component<any, undefined> {
         ));
     }
 
+    public cleanTabFile(files: File[]): File[] {
+        const tab: File[] = [];
+
+        files.map((file) => {
```

Comment:
Do not use `Array.map()` if the code does not use the return value of the function call. Instead, iterate the list of files using `Array.forEach()` or `for (const item of array) { ... }`.